### DISCUSSION TITLE
PDF preview possible?

### DISCUSSION THREAD

**Discussion by jniggemann** [filebrowser/filebrowser#1623] [Ideas]
Hi is it possible to show previews of non-image files?

I'm thinking about showing the first page of a PDF, for example...

**Comment by jniggemann** [comment]
Oh it's right there, I just hadn't adjusted NoScript correctly.
Nevermind, please close this.

**Reply by mckaygerhard** [reply]
there's a problem with pdf preview.. check https://github.com/filebrowser/filebrowser/issues/1442 and  https://github.com/filebrowser/filebrowser/issues/1201
